homeassistant/components/recorder/core.py

```diff
@@ -719,6 +719,15 @@ def _run(self) -> None:
         if schema_status is None:
             # Give up if we could not validate the schema
             return
+        if schema_status.current_version > SCHEMA_VERSION:
+            _LOGGER.error(
+                "The database schema version %s is newer than the installed version of "
```

Comment:
```suggestion
                "The database schema version %s is newer than what the installed version of "
```
I think it the sentence reads a bit weird without `what` since it ends in `supports`